doc/source/programs/gdal_contour.rst

```diff
@@ -122,6 +122,15 @@ be on the right, i.e. a line string goes clockwise around a top.
 
     Generate contour polygons rather than contour lines.
 
+    When this mode is selected the polygons are created for values between
+    each level specified by :option:`-i` or :option:`-fl` or :option:`-e`,
+    in case :option:`-fl` is used alone at least two fixed levels must be specified.
+
+    The minimum and maximum values from the raster are not automatically added to
+    the fixed levels list but the special values `MIN`` and `MAX`` (case insensitive)
+    can be used to include them.
+
```

Comment:
... but when option `-i` is used together with `-p`, the minimum and maximum values from the raster ARE automatically added, and there is no way to get rid of them.

I know this is not a common use case, but there can be lot of writing to the `-fl` if the DEM has a large height range.